python/tvm/contrib/cc.py

```diff
@@ -7,7 +7,8 @@
 def create_shared(output,
```

Comment:
Maybe it is easier to create a ndk.py to do the actual cross compile to android? We can bundle quite a lot of things into it, like default compiler search.